src/main/java/org/wikivoyage/listings/validators/WebsiteURLValidator.java

```diff
@@ -1,24 +1,50 @@
 package org.wikivoyage.listings.validators;
 
-import org.apache.commons.validator.routines.UrlValidator;
-import org.wikivoyage.listings.entity.Listing;
-
+import java.io.IOException;
+import java.net.HttpURLConnection;
 import java.net.MalformedURLException;
 import java.net.URI;
 import java.net.URISyntaxException;
 import java.net.URL;
 
+import org.apache.commons.validator.routines.UrlValidator;
+import org.wikivoyage.listings.entity.Listing;
+
 public class WebsiteURLValidator extends SimpleValidator {
     @Override
     public void validate(Listing poi) {
         if (poi.getUrl() != null && !poi.getUrl().equals("")) {
-            if (!validWebsiteURL(poi.getUrl())) {
+            if (!validWebsiteURL(poi.getUrl()) || !urlExists(poi.getUrl())) {
                 poi.add(ValidationIssue.INVALID_URL);
             }
         }
     }
 
-    private boolean validWebsiteURL(String urlString) {
+    public boolean urlExists(String urlString) {
```

Comment:
Could you please add this:
```
if (urlString.contains("{{dead link")) {
    return false;
}
```
Thanks! :-)

Reply:
Changes made